pkg/p2p/kadcast/config.go

```diff
@@ -17,3 +17,15 @@ var DefaultAlphaClosestNodes int = 1
 
 // DefaultKNumber is the K number of peers that a node will send on a `FIND_NODES` message
 var DefaultKNumber int = 20
+
+const (
+	// Message types over UDP
+
+	pingMsg      = 0
+	pongMsg      = 1
+	findNodesMsg = 2
+	nodesMsg     = 3
+
+	// Message types over TCP
+	broadcastMsg = 10
```

Comment:
Not sure, but do we need to differenciate between packed forwarding messages and consensus-related messages?

Reply:
Would you elaborate a bit? What do you mean by consensus-related messages and packed forwarding messages

Reply:
So for example. In a consensus round, bidders need to propagate them bids as well as block generators need to propagate them purposed blocks.

These messages should also have a type?

Reply:
These type of messages are already declared within gossip frame which is embedded in TCP packet payload.

Reply:
The messages declared in `pkg/p2p/kadcast/config.go ` are transport-related only (closely bound to kadcast protocol)